operations/research-and-development/product/release-process/cloud-release.md

```diff
@@ -0,0 +1,33 @@
+# Cloud Release Process
+
+This document outlines the Cloud release process for the Mattermost core team. Our release process draws from many best practice Agile, Scrum, and Software Development Lifecycle methodologies.
+
+## Release Timeline
+
+Notes:
+- All cut-off dates are based on 10am in San Francisco [(Pacific Time](http://everytimezone.com/)) on the day stated.
+- T-minus counts are measured in "working days" (weekdays other than major holidays concurrent in US and Canada) prior to release day.
+
+### A. (T-minus 8 working days) Fast-forward master to cloud branch
+
+1. Release Manager:
+    - Ask Cloud SRE team to update the community-release server to the same version as the Mattermost Cloud test installations.
+    - Confirm all Jira tickets are resolved for QA testing.
+    - Ask QA to do high priority testing.
```

Comment:
With the new schedule wouldn't this be on t-minus 8 days? e.g. start testing on the Tuesday and then not release until next Wednesday

Reply:
I clarified the text a bit to "Notify QA that the test servers are ready for QA testing. Normally QA testing starts on the next day (at T-8)."